src/contract_wrappers/exchange_wrapper.ts

```diff
@@ -482,36 +491,36 @@ export class ExchangeWrapper extends ContractWrapper {
      */
     @decorators.contractCallErrorHandler
     public async cancelOrderAsync(
-        order: Order|SignedOrder, takerTokenCancelAmount: BigNumber.BigNumber): Promise<BigNumber.BigNumber> {
+        order: Order|SignedOrder, canceltakerTokenAmount: BigNumber.BigNumber): Promise<BigNumber.BigNumber> {
```

Comment:
Also, I know there was a typo in the smart contract, but we shouldn't propagate that error to 0x.js. Here, we should use the appropriate camel-casing: `cancelTakerTokenAmount`.